neurons/validator/validator.py

```diff
@@ -107,14 +115,14 @@ def __init__(self):
         )
 
         # Init device.
-        self.device = torch.device(self.config.alchemy.device)
+        self.device = get_device(torch.device(self.config.alchemy.device))
```

Comment:
It seems like most (maybe all?) calls to device have been replaced with just get_device() so do we still need self.device at all?